README.md

```diff
@@ -1,2 +1,163 @@
-# To-Do-List
-To Do List is a project that lets users manage a simple list of tasks.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="images/PSLogo.svg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>TO DO LIST</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 To Do List <a name="about-project"></a>
+
+This project is coded with standard languages like HTML, CSS and JavaScript.
+
+**To Do List** is a project that let users to manage a simple list of tasks. You can see a hard coded list of tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.spec.whatwg.org/multipage/">HTML 5 </a></li>
+    <li><a href="https://html.spec.whatwg.org/multipage/">CSS 3 </a></li>
+    <li><a href="https://html.spec.whatwg.org/multipage/">JavaScript </i></a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **You can see a hard code list of tasks**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://elafildecolor.github.io/ToDoList/dist/index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+In order to run this project you need:
+
+- Git or Github Desktop
+
+- Code editor like: [Visual Studio code](https://code.visualstudio.com/), [Sublime text](https://www.sublimetext.com/) or others...
+
+- Npm and Node.js
+
+### Setup
+
+- You can download the **Zip** file on the GitHub repository, or clone the repository with:
+```
+git clone https://github.com/elafildecolor/To-Do-List.git
+```
+
+- Access the cloned directory with:
+```
+cd MyPortfolio
```

Comment:
Great job @elafildecolor 🤝

- [ ] [Optional] You may kindly update this line as well by replacing `MyPortfolio` with `To-Do-List`.